docs/Reference/Pantheon-API-Methods.md

```diff
@@ -3771,6 +3771,37 @@ None
     The `EEA` API methods are not enabled by default for JSON-RPC. Use the [`--rpc-http-api`](Pantheon-CLI-Syntax.md#rpc-http-api) 
     or [`--rpc-ws-api`](Pantheon-CLI-Syntax.md#rpc-ws-api) options to enable the `EEA` API methods.
 
+### eea_getTransactionCount
+
+Returns the private transaction count for specified account and privacy group. 
+
+**Parameters** 
+
+`data` - Account address
+
+`data` - Privacy group ID 
```

Comment:
Is this right? Both parameters are named 'data'. I'm not familiar enough with these APIs to make any type of suggestion though.

Reply:
They're the parameter types.  The JSON-RPC parameters aren't named